### ISSUE TITLE
Feature: list objects by their associations and/or see "deeper" associations

### ISSUE THREAD

**Issue by ianballou** [pulp/pulp-cli#132]
It would be nice to see object associations without having to search around.

Example: See publications and distributions associated with a repo.  For now you have to list the distributions, see what publications are associated, then look at the publication to see what repositories are associated.

Another example in a different direction: list repositories from a publication.

From a Katello developer's perspective, we do this all the time from the Rails console. Pretty much all of the associations you could wish for are there and it really helps with debugging.

**Comment by mdellweg** [comment]
can you elaborate, how you'd expect such a command to look like? Would it be a bunch of extra options for `show` ?

**Comment by ianballou** [comment]
I'm revisiting this since Pulpcon is coming soon. I think perhaps a search query could serve the purpose well.

**Comment by mchithu** [comment]
Yes, repo/remote/worker name lookup in the task query will help us much.

**Comment by mdellweg** [comment]
You can search by both worker and used resources of a task.
Also there is a task subcommand for repositories. `pulp file repository task list --help`
